ruby_on_rails/app_initialisation.md

```diff
@@ -109,7 +109,7 @@ create the initializer for figaro in `config/initializers/figaro.rb`:
 
   ```ruby
   config.force_ssl = true # uncomment
-  config.log_level = ENV['RAILS_LOG_LEVEL']&.to_sym || :warn # change
+  config.log_level = ENV.fetch("RAILS_LOG_LEVEL", "warn") # change
```

Comment:
Did you mean to change this from a symbol to a string? It's unlikely to matter but the official docs do use symbols: https://guides.rubyonrails.org/debugging_rails_applications.html#log-levels

Reply:
Well the generated code when creating a new rails 7 project had `ENV.fetch("RAILS_LOG_LEVEL", "info")`

Reply:
Ok, in that case the docs are probably outdated.